airbyte-integrations/bases/base-java/src/main/resources/AirbyteLogMessageTemplate.json

```diff
@@ -0,0 +1,13 @@
+{
+  "type": "LOG",
+  "log": {
+    "level": {
+      "$resolver": "level",
+      "field": "name"
+    },
+    "message": {
```

Comment:
Looking at the documentation for JsonTemplateLayout it seems that it should be possible to log stack trace w/o requiring that we concatenate `logMessage` and `ExceptionUtils.getStackTrace(e)`: https://logging.apache.org/log4j/2.x/manual/json-template-layout.html